tslib/src/azure/app-insights.ts

```diff
@@ -1,32 +1,48 @@
-import * as appInsights from 'applicationinsights'
 import config from '../config'
+import { isNil } from 'ramda'
+const appInsights = require('applicationinsights')
+
+const cloudRoleName = 'TsLib'
```

Comment:
this will get handed to all the functions.  i see there is a specific one for the pupil-api, but i wonder if we can override it for each function?  Out of scope for this PR i would say....

Reply:
This is only a default.  There is an override available by passing the cloudRole string into the startAppInsightsIfConfigured()` func.